src/main/java/com/Autopark/servlets/ViewDiagnosticServlet.java

```diff
@@ -0,0 +1,65 @@
+package com.Autopark.servlets;
+
+import com.Autopark.Auto.AutoCheck;
+import com.Autopark.dtoFacade.DtoService;
+import com.Autopark.infrastructure.configuratots.ObjectConfigurator;
+import com.Autopark.infrastructure.configuratots.impl.AutowiredObjectConfigurator;
+import com.Autopark.infrastructure.core.annotations.Autowired;
+import com.Autopark.infrastructure.core.impl.ApplicationContext;
+import com.Autopark.infrastructure.orm.EntityManager;
+import com.Autopark.infrastructure.orm.service.PostgreDataBaseService;
+import com.Autopark.parser.ParserVehicleFromDB;
+import com.Autopark.parser.ParserVehicleInterface;
+import com.Autopark.repairAuto.Fixer;
+import com.Autopark.repairAuto.MechanicService;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@WebServlet("/viewDiagnostic")
+public class ViewDiagnosticServlet extends HttpServlet {
+
+    private DtoService vehicleTypeService;
```

Comment:
please, use @Autowired functionality which you create

Reply:
fixed